### PR TITLE
CIRC-736 update service point for using calendar

### PR DESCRIPTION
**Pull request by roman-barannyk**
## Purpose
Change service point for calendar using in calculation minutes for overdue fines.  According to the requirements it should be the service point hours for the primary service point of the effective location of the item.

Resolves [CIRC-736](https://issues.folio.org/browse/CIRC-736)

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

### CODE REVIEW

File: src/main/java/org/folio/circulation/domain/OverduePeriodCalculatorService.java [hunk 1]
```diff
@@ -153,4 +155,7 @@ private int getGracePeriodMinutes(Loan loan) {
     return loan.getLoanPolicy().getGracePeriod().toMinutes();
   }
 
+  private UUID getOwnerServicePoint(Loan loan) {
```
Comment (alexanderkurash) [inline src/main/java/org/folio/circulation/domain/OverduePeriodCalculatorService.java:158]: Maybe this method should be renamed to something like getItemLocationPrimaryServicePoint? The fact that it's the same service point that we use for determining owner is probably not a coincidence, but this class knows nothing about fee/fine owners.

File: src/main/java/org/folio/circulation/domain/OverduePeriodCalculatorService.java [hunk 2]
```diff
@@ -61,8 +62,9 @@ boolean preconditionsAreMet(Loan loan, DateTime systemTime, Boolean shouldCountC
 
   private CompletableFuture<Result<Integer>> minutesOverdueExcludingClosedPeriods(Loan loan, DateTime returnDate) {
     DateTime dueDate = loan.getDueDate();
+    String ownerServicePoint = getOwnerServicePoint(loan).toString();
```
Comment (alexanderkurash) [inline src/main/java/org/folio/circulation/domain/OverduePeriodCalculatorService.java:65]: If method is renamed, the variable should be renamed too.

File: src/main/java/org/folio/circulation/domain/OverduePeriodCalculatorService.java (hunk 1 above)
Comment (roman-barannyk) [reply src/main/java/org/folio/circulation/domain/OverduePeriodCalculatorService.java]: @alexanderkurash, 
Thanks, renamed

File: src/main/java/org/folio/circulation/domain/OverduePeriodCalculatorService.java (hunk 2 above)
Comment (roman-barannyk) [reply src/main/java/org/folio/circulation/domain/OverduePeriodCalculatorService.java]: renamed

**Pull request merged**